include/tvm/tir/stmt.h

```diff
@@ -802,6 +802,9 @@ class ForNode : public StmtNode {
   ForKind kind;
   /*! \brief The body of the for loop. */
   Stmt body;
+  /*! \brief The additional termination condition of the for loop. */
+  Optional<PrimExpr> test;
```

Comment:
It would be helpful to have a RFC discussion, since different strategies changes to the IR can have different implications